jobs/JAQM_MAKE_ICS

```diff
@@ -8,21 +8,11 @@ set -xue
 export USHaqm=${EXECaqm:-${HOMEaqm}/ush}
 export PARMaqm=${PARMaqm:-${HOMEaqm}/parm}
 
-export subcyc='00'
-export SLASH_ENSMEM_SUBDIR=''
-export ENSMEM_INDX='##'
-
```

Comment:
Pulling out all the SRW stuff that get's in the way -- these aren't needed for Production AQM and can serve as red-herrings when debugging.